src/core/voices/voice_manager.hpp

```diff
@@ -272,8 +274,10 @@ namespace otto::core::voices {
   private:
     std::vector<float> detune_values;
     std::vector<float> rand_values;
-    // Random values. 100% random, organic and fresh.
-    std::array<float, 6> rand_max = {0.94, 0.999, 1.03, 1.06, 0.92, 1.01};
+    // Random values. 100% random, organic and fresh. Works for up to 12 voices.
+    std::vector<float> rand_max_length = {0.94, 0.999, 1.03, 1.06, 0.92, 1.01,
+                                                  1.02, 0.98, 1.0, 1.09, 0.94, 1.05};
```

Comment:
Nope, get those back in a fixed size array of size NumberOfVoices, and don't use the rand_max_length member just for initializing the second one. You can initialize it using a lambda if you want.

Reply:
I am a bit unsure how to do this. I want it to be the same numbers every time, so it is somewhat predictable. Could you give an example?

Reply:
It is now just an array with 12 elements. Inelegant, but will work as long as the voice count is not above that.